tests/InstanceTests.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Linq;
+using FluentAssertions;
+using Xunit;
+
+namespace Wasmtime.Tests
+{
+    public class InstanceFixture
+        : ModuleFixture
+    {
+        protected override string ModuleFileName => "Hello.wat";
```

Comment:
> Linux CI seems to be failing due to a file not copying into the test environment properly. I'm not sure why :/

I think that's a casing issue: The actual file name is `hello.wat`, but is referred to as `Hello.wat` in the code.

```suggestion
        protected override string ModuleFileName => "hello.wat";
```

Reply:
Well spotted, thanks!